plugins/mcreator-localization/lang/texts.properties

```diff
@@ -440,21 +440,19 @@ dialog.preferences.themes.list=<html><br><font style="font-size: 14px;">List of
 dialog.preferences.description=<html><font style="font-size: 16px;">{0}</big><br><div width=700><font style="font-size: 9px; color: gray;">{1}
 dialog.preferences.entry_description=<html>{0}:<br><font style="font-size: 8px; color: gray;">{1}
 dialog.preferences.unknown_property_type={0}: unknown property type
-dialog.search_usages.list=List of mod elements that use at least one {0} from the selected list:
-dialog.search_usages.list.empty=There are no mod elements using any {0} from the selected list.
+dialog.search_usages.list=List of mod elements using selected {0}s:
+dialog.search_usages.list.empty=No mod elements using any of the selected {0}s found.
 dialog.search_usages.list.item=<html>{0}<br><small>Type: {1}
 dialog.search_usages.open_selected=Edit selected mod element
-dialog.search_usages.title=Usages list
-dialog.search_usages.deletion.confirm_msg=<html><p align=center>Some elements from the selected {0} list are still used by mod elements listed below.<br>If you proceed, the workspace might get broken!<br><br><b>Are you sure that you want to delete selected elements?
-dialog.search_usages.deletion_safe.confirm_msg=<html>There are no mod elements using any {0} from the selected list.<br>However, they may still be used outside the workspace (by other mods). If you proceed,<br>these references will become invalid which might lead to broken resources or game crashes.<br><br>Are you sure that you want to delete selected elements?
+dialog.search_usages.title=Element usages
```

Comment:
Are you sure the word **element** is appropriate here?

Reply:
You have used this term too:

![image](https://github.com/MCreator/MCreator/assets/16374228/59a42233-1e99-4e30-bddf-3f0f6dc1e7bd)

Reply:
List is made out of elements in other meaning. Otherwise I'd come up with this title first.

Reply:
> A more suitable name is ofc welcome

But you need to keep in mind this title appears in both usages and deletion dialogs

Reply:
> Fair enough.